index.js

```diff
@@ -106,7 +106,8 @@
 
           var fn = this.__outsideClickHandler = generateOutsideCheck(
             ReactDOM.findDOMNode(instance),
-            instance.handleClickOutside.bind(instance),
+            instance.handleClickOutside,
```

Comment:
We might as well only pass in the instance, then, and the refer to `instance.handleClickOutside` rather than `eventHandler` inside the `generateOutsideCheck` code.